controllers/v1/surveysController.js

```diff
@@ -0,0 +1,709 @@
+/**
+ * name : surveysController.js
+ * author : Deepa
+ * created-date : 06-Sep-2020
+ * Description : Surveys information.
+ */
+
+// Dependencies
+const csv = require("csvtojson");
+const FileStream = require(ROOT_PATH + "/generics/fileStream");
+const surveysHelper = require(MODULES_BASE_PATH + "/surveys/helper");
+const observationsHelper = require(MODULES_BASE_PATH + "/observations/helper");
```

Comment:
@deepakr123 - Remove this dependency on observation helper.

Reply:
resolved